rfcs/0168-Trigger-Tests-Based-on-PR-Comments.md

```diff
@@ -0,0 +1,31 @@
+# RFC 168 - Allow comments to trigger tasks for non-collaborators
+* Comments: [#168](https://github.com/taskcluster/taskcluster-rfcs/pull/168)
+* Proposed by: @bhearsum
+
+# Summary
+
+Allow collaborators to a Github repository to request that Tasks be run on Pull Requests that ran no tasks due to a restrictive `pullRequests` policy.
+
+## Motivation
```

Comment:
It would be nice to have a way to trigger tasks from pull request comments also for cases where there are some tasks that are pretty expensive to run and are not supposed to run for every PR.
For example, in https://github.com/mozilla/rust-code-analysis we have a task that runs some analysis on mozilla-central and takes more than 20 minutes. It is useless to run it on every PR, and it is not feasible to write a rule to decide when to run the analysis: a human must decide when it's worth doing it. Currently, we push a commit to run it, and then remove it before landing. It would be much better if we could trigger a task by writing a comment in the PR.

A similar thing happens in pdf.js (it doesn't currently use Taskcluster but a custom CI, and the CI runs on comments).

Reply:
Closing this because we've discussed a solution for this use case further down.